lib/runctl.js

```diff
@@ -31,7 +31,7 @@ var server = require('strong-control-channel/server').create(onRequest);
 // Expose runctl using node IPC only in the master/supervisor (workers
 // attach targetctl to the master).
 var ipcctl = null;
-if (cluster.isMaster && process.send && !process.env.SL_RUN_SKIP_IPCCTL) {
+if (cluster.isMaster && !process.env.SL_RUN_SKIP_IPCCTL) {
```

Comment:
this tries to create the process ipcctl channel in standalone mode... doest that have no ill effect?

I think you might need

```
if cluster.isMaster && !skip:
  if wsUrl
    ipcctl = wsChannel()
  elseif process.send
   processChannel()
```